packages/lib-classifier/src/store/ClassificationStore.js

```diff
@@ -118,6 +118,8 @@ const ClassificationStore = types
       if (classification) {
         const annotation = classification.annotations.get(task.taskKey) || createDefaultAnnotation(task)
         annotation.value = annotationValue
+        const feedback = getRoot(self).feedback
+        if (feedback.isActive) feedback.update(annotation)
```

Comment:
I'm not a fan of `if`s without blocks

Reply:
I'd drop this completely and expect `feedback.update()` to make any checks like this.

Reply:
Alternatively, wire up `onAction` to fire `feedback.update` on `createDefaultAnnotation`..